components/panel/header.js

```diff
@@ -0,0 +1,9 @@
+function PanelHeader( { children } ) {
+	return (
+		<div className="components-panel__header">
```

Comment:
Seems odd we push responsibility of wrapping `strong` to the rendering component instead of baking in consistent behavior to the `PanelHeader` itself. I see we have a need to exempt icons from bold effect, but maybe this could be achieved with a separate prop instead?